src/state/apis/nft/nftApi.ts

```diff
@@ -28,16 +29,40 @@ const moduleLogger = logger.child({ namespace: ['nftApi'] })
 
 type NftState = {
   selectedNftAvatarByWalletId: Record<WalletId, AssetId>
+  nfts: {
+    byId: Record<AssetId, NftItem>
```

Comment:
may want to make these `PartialRecord`'s a la the asset slice https://github.com/shapeshift/web/blob/develop/src/state/slices/assetsSlice/assetsSlice.ts#L36

Reply:
I like safety ae260b0ddc3ae3fd3c895533ef585b963190dec8